src/Modal/__story__/story.tsx

```diff
@@ -0,0 +1,54 @@
+import {createElement as h} from 'react';
+import {storiesOf} from '@storybook/react';
+import {action} from '@storybook/addon-actions';
+import {linkTo} from '@storybook/addon-links';
+import {Modal} from '..';
+import ShowDocs from '../../../.storybook/ShowDocs'
+
+storiesOf('UI/Modal', module)
+  .add('Documentation', () => h(ShowDocs, {md: require('../../../docs/en/Modal.md')}))
```

Comment:
are you using require here to avoid linting errors?

Reply:
No, because `.md` is not a module, and TypeScript will not `import` it.